docs/unified-test-documentation/dasharo-compatibility/31F-audio-subsystem.md

```diff
@@ -39,9 +37,9 @@ and can be detected from the operating system.
 1. Log into the system by using the proper login and password.
 1. Open a terminal window and execute the following command:
 
-```bash
-cat /sys/class/sound/card0/hwC0D*/chip_name
-```
+    ```bash
+    cat /sys/class/sound/card0/hwC0D*/chip_name
+    ```
```

Comment:
In the `Expected result` section we should add information, that the below output is only `Example output`. Depending on mounted devices, the output might be different.

Reply:
added more information